rpc/internal/perf_service.go

```diff
@@ -56,6 +56,14 @@ func (srv *perfService) CreateMetricSeries(ctx context.Context, result *ResultDa
 	if err := record.SaveNew(ctx); err != nil {
 		return resp, newRPCError(codes.Internal, errors.Wrap(err, "problem saving record"))
 	}
+	if len(record.Rollups.Stats) > 0 && len(record.Artifacts) == 0 {
```

Comment:
In theory someone could add a non-ftdc artifact and rollups, which, with this if statement, would never create an update-time-series job. It's fine to call signal processing more than once with updated data, so this should just be `if len(record.Rollups.Stats) > 0`.

Reply:
done.
Something is confusing me about how it works: what prevents us from enqueueing the FTDC rollups job for non-FTDC artifacts? Is it [this schema check](https://github.com/evergreen-ci/cedar/blob/32e8b7a079bd8ee011244017ef027e6d36efa2fa/rpc/internal/perf_service.go#L255)? In other words, is the raw-events schema particular to FTDC artifacts?

Reply:
The schema `raw-events` represents a (performance point)[https://github.com/mongodb/ftdc/blob/main/events/performance.go] which cedar understands in [perf/ftdc_rollups.go](https://github.com/evergreen-ci/cedar/blob/main/perf/ftdc_rollups.go). Any data with another schema passed to the rollups job would fail. 

Though, I just realized that we do not check the file format in (addFTDCRollupsJob)[https://github.com/evergreen-ci/cedar/blob/main/rpc/internal/perf_service.go#L249]—this currently is not a problem cause no one is attaching non-ftdc files. But there should be a check that `artifact.Format == model.FormatFTDC`.

Reply:
I opened https://jira.mongodb.org/browse/EVG-15372